tsconfig-strict.json

```diff
@@ -18,7 +18,6 @@
     "baseUrl": ".",
     "paths": {
       "app.constants": ["core/templates/app.constants"],
-      "app-type.constants": ["core/templates/app-type.constants"],
```

Comment:
Not related to migration. But removed this file and forgot to remove this in my previous pr.